### ISSUE TITLE
Need explanation of base URL setting

### ISSUE THREAD

**Issue by patricorgi** [darmiel/raycast-extension-jellyfin#3]
I pretty much run Jellyfin in LAN with default settings. So for me the URL to Jellyfin Web UI is just `http://localhost:8096`. 

I have followed the provided instruction to get my user id and API key, but I have trouble setting up the base URL.

I have tried enter following URLs in the plugin settings but none of them work. 

- `http://localhost:8096`
- `http://localhost:8096/web/index.html`
- `http://localhost:8096/myJellyfin/web/index.html` after setting Base URL in Jellyfin settings to `myJellyfin`

**Comment by darmiel** [comment]
It should be `http://localhost:8096` - is there any error?

**Comment by patricorgi** [comment]
Thanks for the swift reply! This is what I see when running `Search Movies and Shows`.
<img width="774" alt="Screenshot 2023-10-08 at 3 48 02 PM" src="https://github.com/darmiel/raycast-extension-jellyfin/assets/15998986/730f2abc-7d8d-44e8-b63e-699c2e8a9653">

**Comment by darmiel** [comment]
That's interesting! Can you check what happens when you call the domain http://localhost:8096/Users/bf... directly? You should get a HTTP 401, is this the case?

**Comment by patricorgi** [comment]
Yep. If I open the url `http://localhost:8096/Users/<userID>` directly in my browser, I got HTTP 401.

**Comment by darmiel** [comment]
is there any way to expand the error? If you know a bit of JavaScript/Raycast development, you could also try to run the addon in development mode which would show us the full stack trace.

**Comment by patricorgi** [comment]
Sorry I don't know JS, and I also don't know how to enable developer mode in Raycast.
